website/static/js/filters.js

```diff
@@ -0,0 +1,124 @@
+(function($){
+    $.fn.filters = function (options) {
+        var settings = $.extend({
+            items: '.items'
```

Comment:
Your default for variable ```items``` is not an array, the ```items.join()``` in line 7 will fail if it uses the default. Can you check this?